### PR TITLE
Kubernetes policy management paper

### PR DESCRIPTION
**Pull request by JimBugwadia**
Signed-off-by: Jim Bugwadia <jim@nirmata.com>

Kubernetes Policy paper. 

Prior PR with review history: https://github.com/kubernetes/community/pull/6140.

### CODE REVIEW

**Comment by PushkarJ** [comment]
Thanks @JimBugwadia . Can we move this whole directory `papers` under `sig-security-docs` ? cc @savitharaghunathan

**Comment by JimBugwadia** [comment]
Preview in [markdown ](https://github.com/kubernetes/sig-security/blob/bffb43a1178767a829921c28696a11bfc9d4fd2e/papers/policy/kubernetes-policy-management.md) and [PDF](https://github.com/kubernetes/sig-security/blob/bffb43a1178767a829921c28696a11bfc9d4fd2e/papers/policy/kubernetes-policy-management.pdf).

**Comment by JimBugwadia** [comment]
@PushkarJ - done!

**Comment by JimBugwadia** [comment]
Previews in [Markdown ](https://github.com/kubernetes/sig-security/blob/009410f830dccd388f30107dee3b40455742278e/sig-security-docs/papers/policy/kubernetes-policy-management.md) and [PDF](https://github.com/kubernetes/sig-security/blob/009410f830dccd388f30107dee3b40455742278e/sig-security-docs/papers/policy/kubernetes-policy-management.pdf).

File: sig-security-docs/papers/policy/kubernetes-policy-management.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+
+Policies are configured using interfaces provided by the PAP which provides multi cluster policy management, enabling policies to be bound to managed clusters. The PAP deploys the policies to the managed clusters based on this binding. A flexible approach of doing so is by associating labels with the clusters and applying policies to clusters that meet certain placement rules derived from these labels.
+
+The PAP  is used for authoring, deploying, and managing policy changes. However, most real world implementations of a policy management system will also provide capabilities to manage compliance mappings, manage policy results, provide workflows for process automation, enable collaboration across teams, and provide integrations into other enterprise systems such as for identity management, version control, and notifications. In addition to managing and distributing policies, a PAP performs several other useful functions like:
```
Comment (celestehorgan) [inline sig-security-docs/papers/policy/kubernetes-policy-management.md:136]:
```suggestion
The PAP is used for authoring, deploying, and managing policy changes. However, most real world implementations of a policy management system will also provide capabilities to manage compliance mappings, manage policy results, provide workflows for process automation, enable collaboration across teams, and provide integrations into other enterprise systems such as for identity management, version control, and notifications. In addition to managing and distributing policies, a PAP performs several other useful functions like:
```

**Review by celestehorgan — COMMENTED** [review]
One tiny change. otherwise LGTM!

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm
Thanks for working on this PR, @JimBugwadia!

File: sig-security-docs/papers/policy/kubernetes-policy-management.md [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+* **Namespaces**: Kubernetes namespaces ([documentation link](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)) allow API level segmentation of resources and provide a way to share clusters across workloads and teams. Even when clusters are dedicated to a team or application, namespaces should be used as a security boundary to restrict cluster-wide access. 
+* **Role Based Access Controls (RBAC)**: Kubernetes allows cluster-wide and namespaced definitions of roles, with granular permissions, and role-bindings to map users or service accounts to roles ([documentation link](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)).
+* **Pod security admission**: Pods are the basic unit of execution in Kubernetes, and each pod has a security context that defines its privileges and other security requirements. The [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) defines three levels of security policies to apply to pods, and [Pod Security Admission Controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/) provides an implementation for applying the policies at a namespace level.
```
Comment (reylejano) [inline sig-security-docs/papers/policy/kubernetes-policy-management.md:169]: nit: case
reference: https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-levels

```suggestion
* **Pod Security admission**: Pods are the basic unit of execution in Kubernetes, and each pod has a security context that defines its privileges and other security requirements. The [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) defines three levels of security policies to apply to pods, and [Pod Security Admission Controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/) provides an implementation for applying the policies at a namespace level.
```

**Comment by PushkarJ** [comment]
/lgtm

(One more time now that prow is up)

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

**Review by savitharaghunathan — COMMENTED** [review]
/approve

**Pull request merged**